src/common/ConfigReader.cpp

```diff
@@ -241,16 +241,20 @@ namespace SDDM {
         if (section) {
             if (entry && !entry->matchesDefault())
                 remainingEntries.insert(section, entry);
-            else
-                for (const ConfigEntryBase *b : section->entries().values())
+            else {
+                const auto entries = section->entries();
```

Comment:
Why a separate variable here?

Reply:
To be systematic.
Since ```section->entries()``` returns `const QMap` there's no need. But by looking at the code, that's not obvious. 
Just to make it more easier to understand, since there's no performance penalty.

Reply:
It's not highlighting that it's returning a `QMap`, so to indicate the `const`, you could just explicitly use `qAsConst` like below?

Reply:
I'm Ok with that.